magma/circuit.py

```diff
@@ -652,6 +652,7 @@ def __new__(metacls, name, bases, dct):
                                 "instead", debug_info=self.debug_info)
                 with _DefinitionContextManager(self._context_):
                     self.definition()
+                self._context_.finalize(self)
```

Comment:
why didn't this cause problems before with inline verilog?

Reply:
Inline verilog is processed in a pass, not during the finalize logic. The file open/display logic is finalized there, but perhaps that wasn't being used (I think they ended up building their own logging library that generates inline verilog)